js/bootstrap-colorpicker-module.js

```diff
@@ -1,95 +1,95 @@
 angular.module('colorpicker.module', [])
-    .factory('Helper', function () {
-      'use strict';
+.factory('Helper', function () {
+  'use strict';
+  return {
+    closestSlider: function (elem) {
+      var matchesSelector = elem.matches || elem.webkitMatchesSelector || elem.mozMatchesSelector || elem.msMatchesSelector;
+      if (matchesSelector.bind(elem)('I')) {
+        return elem.parentNode;
+      }
+      return elem;
+    },
+    getOffset: function (elem, fixedPosition) {
+      var
```

Comment:
This is almost unreadable... if we want one `var` for multiple lines of variables, we should at least do:

```
var x = 0,
    y = 0,
    scrollX = 0,
    scrollY = 0
```